src/Components/App/App.js

```diff
@@ -30,21 +30,36 @@ class App extends Component {
     console.log(currentUser)
   }
 
+  toggleFavourites = (id) => {
+    let { movies, currentUser, favourite } = this.props;
+    let movie =  movies.find(movie => movie.id === id);
+    if (favourite === true) {
```

Comment:
Bless your soul, Sam. This is great!! idk why i didn't think about this last night!! thank you!!